docs/Installation.md

```diff
@@ -1,15 +1,25 @@
 # Installation Guide
 
-## Introduction
+## Choosing a setup
 
 There are several ways to install AzerothCore, you need to choose **ONE**.
 
+### Full (supported)
+
+These are the officially-supported and complete ways of installing AzerothCore, for any purposes.
```

Comment:
Azerothcore Classic setup would land under this too.

Reply:
of course, it does

Reply:
oh my bad, I thought of something else. Ignore it :D